linux_os/guide/system/software/gnome/dconf_db_up_to_date/rule.yml

```
@@ -32,6 +32,8 @@ references:
     cis@rhel7: 1.7.2
     cis@rhel8: 1.8.2
     cis@rhel9: 1.8.2
+    cis@sle12: 1.10
+    cis@sle15: 1.10
```

```suggestion
    cis@sle12: "1.10"
    cis@sle15: "1.10"
```

It's unfortunately interpreting as a number instead of string. Enclosing with double quotes should be enough.

---

Thanks @ggbecker :bow: